packages/webpack/src/utils/createTemplate.js

```diff
@@ -51,9 +51,15 @@ function createRefreshTemplate(source, chunk, hash, mainTemplate) {
 
 	// Webpack generates this line whenever the mainTemplate is called
 	const moduleInitializationLineNumber = lines.findIndex(line =>
-		line.startsWith('modules[moduleId].call')
+		options.runsInNextJs
```

Comment:
@JoviDeCroock maybe you can add a comment why this condition is needed. I still did not get the point where this originates from 100%